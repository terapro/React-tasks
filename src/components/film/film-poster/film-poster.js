import React from 'react';
import PropTypes from 'prop-types';

const FilmPoster = (props) => (
  <div className="film-poster">
    <img src={props.posterPath} />
    </div>
);

FilmPoster.propTypes ={
  posterPath: PropTypes.string
};




export {FilmPoster};