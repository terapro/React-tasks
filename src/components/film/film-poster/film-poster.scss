.film-poster {

}

.film-poster img{
  width: $filmPosterWidth;


}