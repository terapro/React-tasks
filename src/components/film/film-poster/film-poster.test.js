import React from 'react';
import {FilmPoster} from 'src/components/film/film-poster/film-poster.js';
import {shallow} from 'enzyme';

const mockPath = 'https://image.tmdb.org/t/p/w500/pU1ULUq8D3iRxl1fdX2lZIzdHuI.jpg';

describe('<FilmPoster />', () => {
  describe('Rendering', () => {
    it('should render the component', () => {
      const wrapper = shallow(<FilmPoster posterPath={mockPath} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});