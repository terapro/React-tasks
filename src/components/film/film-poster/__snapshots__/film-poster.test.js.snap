// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmPoster /> Rendering should render the component 1`] = `
<div
  className="film-poster"
>
  <img
    src="https://image.tmdb.org/t/p/w500/pU1ULUq8D3iRxl1fdX2lZIzdHuI.jpg"
  />
</div>
`;
